activation/poetdb.go

```diff
@@ -127,16 +136,20 @@ func (db *PoetDb) publishPoetProofRef(key poetProofKey, poetProofRef []byte) {
 	delete(db.poetProofRefSubscriptions, key)
 }
 
-func (db *PoetDb) GetMembershipByPoetProofRef(poetRef []byte) (map[common.Hash]bool, error) {
-	poetProofBytes, err := db.store.Get(poetRef)
+func (db *PoetDb) GetProofMessage(proofRef []byte) ([]byte, error) {
+	return db.store.Get(proofRef)
```

Comment:
could this be called in a go routine? shoould we lock here if so?
Have you run this code with --race?

Reply:
while `--race` breaks many of our unit tests (23 of them), it doesn't look like any of the failures are related to the changes in this PR.